woocommerce/woocommerce
# Reduce CYS intro screen loading time

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/40793

Since we are redirecting to the assembler hub through a full redirect or showing it in an iframe, we no longer need to load the resources that are required by the assembler hub in the intro screen. This PR reverts the changes in https://github.com/woocommerce/woocommerce/pull/39843#discussion_r1301168806 and updates the `possibly_add_site_editor_scripts` logic to reduce the loading time.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new WooCommerce installation with this version. 
2. Install  WooCommerce Block [Nightly](https://github.com/woocommerce/woocommerce-blocks/releases/tag/nightly)
3. Make sure to enable `customize-store` feature flag
3. Navigate to the CYS task
4. Observe that the Intro screen loads much, much faster than before




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
